docs/src/env-vars.md

```diff
@@ -99,3 +99,31 @@ If not defined (default) Application Default Credentials are used.
 #### Values accepted
 
 String
+
+### RUNAI_STREAMER_DIST
+
+Enables distributed streaming for multiple devices
+
+If not defined (default) distributed streaming is enabled only for streaming from object storage to CUDA devices
+
+### Values accepted
+
+Boolean `0` or `1`
+
+#### Default value
+
+None
```

Comment:
Its a good practice to have a value of "1", "0", or "auto". And auto would be the default.
That way the default value would be one of the accepted values